utilities/rpp-unittests/HOST_NEW/testAllScript.sh

```diff
@@ -110,7 +110,7 @@ if [[ "$1" -lt 0 ]] | [[ "$1" -gt 81 ]]; then
     exit 1
 fi
 
-if [[ "$2" -lt 0 ]] | [[ "$2" -gt 81 ]]; then
+if [[ "$2" -lt 0 ]] | [[ "$2" -gt 82 ]]; then
     echo "The ending case# must be in the 0-81 range!"
```

Comment:
Similar change at L114, L119, L120 to correspond with 0-82 range.
L132, L133 too